Assets/VRTK/Scripts/VRTK_AdaptiveQuality.cs

```diff
@@ -0,0 +1,708 @@
+    /// </list>
+    /// <para />
+    /// This script currently changes the following to reach these goals:
+    /// <list type="bullet">
+    /// <item> <description>Rendering resolution and viewport (aka Dynamic Resolution)</description> </item>
+    /// </list>
+    /// <para />
+    /// In the future it could be changed to also change the following:
+    /// <list type="bullet">
+    /// <item> <description>MSAA level</description> </item>
+    /// <item> <description>Fixed Foveated Rendering</description> </item>
+    /// <item> <description>Radial Density Masking</description> </item>
+    /// <item> <description>(Non-fixed) Foveated Rendering (once HMDs support eye tracking)</description> </item>
+    /// <item> <description>...</description> </item>
+    /// </list>
+    /// </remarks>
+    /// <example>
+    /// `VRTK/Examples/039_CameraRig_AdaptiveQuality` displays the frames per second in the centre of the headset view.
+    /// The debug visualization of this script is displayed near the top edge of the headset view.
+    /// Pressing the trigger generates a new sphere and pressing the touchpad generates ten new spheres.
+    /// Eventually when lots of spheres are present the FPS will drop and demonstrate the script.
+    /// </example>
+    public sealed class VRTK_AdaptiveQuality : MonoBehaviour
+    {
+        #region Public fields
+
+        [Tooltip("Toggles whether the render quality is dynamically adjusted to maintain VR framerate.\n\n"
+                 + "If unchecked, the renderer will render at the recommended resolution provided by the current VRDevice.")]
+        public bool active = true;
+
+        [Tooltip("Toggles whether to show the debug overlay.\n\n"
+                 + "Each square represents a different level on the quality scale. Levels increase from left to right,"
+                 + " and the first green box that is lit above represents the recommended render target resolution provided by"
+                 + " the current VRDevice. The yellow boxes represent resolutions below the recommended render target resolution.\n"
+                 + "The currently lit box becomes red whenever the user is likely seeing reprojection in the HMD since the"
+                 + " application isn't maintaining VR framerate. If lit, the box all the way on the left is almost always lit red because"
+                 + " it represents the lowest render scale with reprojection on.")]
+        public bool drawDebugVisualization;
+
+        [Tooltip("Toggles whether to allow keyboard shortcuts to control this script.\n\n"
+                 + "The supported shortcuts are:\n"
+                 + "- Shift+F1: Toggle debug visualization on/off\n"
+                 + "- Shift+F2: Toggle usage of override render scale on/off\n"
+                 + "- Shift+F3: Decrease override render scale level\n"
+                 + "- Shift+F4: Increase override render scale level")]
```

Comment:
Use markdown to create this list within the tooltip like so:

```
        [Tooltip("Toggles whether to allow keyboard shortcuts to control this script.\n\n"
                 + "* The supported shortcuts are:\n"
                 + " * Shift+F1: Toggle debug visualization on/off\n"
                 + " * Shift+F2: Toggle usage of override render scale on/off\n"
                 + " * Shift+F3: Decrease override render scale level\n"
                 + " * Shift+F4: Increase override render scale level")]
```

Then it will generate the list in markdown correctly.